### ISSUE TITLE
Indigo support

### ISSUE THREAD

**Issue by vrabaud** [allenh1/ros-overlay#1]
Hi, thx for your work, it's very clean and well documented ! I don't know much about Gentoo: would your instructions also work with Indigo ?

**Comment by vrabaud** [comment]
Actually, how is that repo related to https://github.com/ros/ros-overlay ?

**Comment by allenh1** [comment]
The one you reference was a similar effort that was made a couple years back. I have used that in cresting this repo.

As for indigo, I definitely think it should work (though have not tested it myself). I will do so as soon as I have a few moments.

**Comment by vrabaud** [comment]
Cool for Indigo !
@tfoote, maybe @allenh1 should have write rights on https://github.com/ros/ros-overlay no ? Configurable at: https://github.com/orgs/ros/teams/gentoo

**Comment by tfoote** [comment]
That seems reasonable @wyc has been maintaining that the most. @allenh1 would you be willing to work with @wyc to merge this into ros/ros-overlay?

**Comment by allenh1** [comment]
Absolutely. There was a wyc-overlay that became the ros/ros-overlay repo. I actually did some setup with layman for the wyc-overlay (the one I started with when rewriting was an old fork of wyc-overlay).

**Comment by wyc** [comment]
sounds great! @allenh1 thanks for your work.

@tfoote would it make sense to add @allenh1 as a member of ros/ros-overlay? as mentioned, we've worked together in the past to bring ros support to gentoo.

**Comment by tfoote** [comment]
Great. 

@allenh1 I've sent you a request to join the gentoo team @vrabaud linked to.

**Comment by allenh1** [comment]
Thank you much! I'll revize documentation and begin moving things as soon as I am able.

**Comment by wyc** [comment]
@allenh1 https://bugs.gentoo.org/show_bug.cgi?id=557764 if you have some time this might be a good thing to address as you're making your improvements.
